src/rt/model.c

```diff
@@ -630,6 +630,21 @@ const void *signal_value(rt_signal_t *s)
    return s->shared.data;
 }
 
+uint32_t signal_size(rt_signal_t *s)
```

Comment:
No need to add `signal_size` and `signal_name` - just include `rt/structs.h` and access the fields directly.